Doc/library/index.rst

```diff
@@ -27,8 +27,8 @@ as a collection of packages, so it may be necessary to use the packaging
 tools provided with the operating system to obtain some or all of the
 optional components.
 
-In addition to the standard library, there is a growing collection of
-several thousand components (from individual programs and modules to
+In addition to the standard library, there is a constantly growing collection of
```

Comment:
instead of constantly growing or growing, i'd go with something such as

"there is an evolving collection of"

Reply:
how about 'active collection of'?

Reply:
that also sounds good.

Reply:
a57f3a1ceddb7187d57b1cd0bc7fa1c2454925a0